parquet-hadoop/src/main/java/org/apache/parquet/format/converter/ParquetMetadataConverter.java

```diff
@@ -259,6 +274,48 @@ public static Statistics toParquetStatistics(org.apache.parquet.column.statistic
     return stats;
   }
 
+  private static boolean shouldIgnoreStatistics(String createdBy) {
+    if (createdBy == null || !createdBy.startsWith("parquet-mr")) {
```

Comment:
they didn't, right? I think we need to handle the case where the createdBy is something like "parquet-mr <sha>" (missing a version number)